update_results.sh

```diff
@@ -0,0 +1,17 @@
+
+# Update the noisy tournaments
+echo "-- Regenerating noisy tournament figures"
+# Run the noisy tournament
+./run_axelrod --noise 0.05 --xc -p 0
+# Copy these figures over, renaming as appropriate
+cp strategies_payoff.png assets/strategies_payoff_noise_5.png
+cp strategies_boxplot.png assets/strategies_boxplot_noise_5.png
+
+# Running noiseless tournament
+echo "-- Running Axelrod, regenerating cache, results, and figures"
+./run_axelrod --rc -p 0
+# Copy the results and figures over to assets
```

Comment:
for the noisy tournament I need the files to be renamed, and I don't necessarily want to overwrite everything. I guess it would be fine for the regular tournament?